packages/fxa-settings/src/components/BentoMenu/index.test.tsx

```diff
@@ -3,41 +3,43 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { render, screen, fireEvent, act } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
 import BentoMenu from '.';
 
 describe('BentoMenu', () => {
   it('renders and toggles as expected with default values', () => {
     render(<BentoMenu />);
 
     const toggleButton = screen.getByTestId('drop-down-bento-menu-toggle');
-    const dropDown = screen.queryByTestId('drop-down-bento-menu');
 
     expect(toggleButton).toHaveAttribute('title', 'Firefox Bento Menu');
-    expect(toggleButton).toHaveAttribute(
-      'aria-controls',
-      'drop-down-bento-menu'
-    );
+    expect(toggleButton).toHaveAttribute('aria-label', 'Firefox Bento Menu');
+    expect(toggleButton).toHaveAttribute('aria-haspopup', 'true');
     expect(toggleButton).toHaveAttribute('aria-expanded', 'false');
-    expect(dropDown).not.toBeInTheDocument();
+    expect(
+      screen.queryByTestId('drop-down-bento-menu')
```

Comment:
Per https://github.com/mozilla/fxa/pull/13240#discussion_r894798671, I wonder if we should do something like:
```js
const dropDownId = 'drop-down-bento-menu';
```

And then we can use a similar pattern like `screen.queryByTestId(dropDownId)` (since we seem to hardcode `'drop-down-bento-menu'` 7&times; in this file).

Reply:
Agreed, I'll create a const for tests that reference it more than once 🙂 .